test/metabase/upload_test.clj

```diff
@@ -97,6 +96,16 @@
   [& body]
   `(do-with-mysql-local-infile-off (fn [] ~@body)))
 
+(defn sync-upload-test-table!
```

Comment:
Awesome idea 🤟 

I'm curious as to how it's faster, and whether there are any gotchas related to that. Perhaps the docstring could metnion what's being skipped?